src/nationGen/misc/Site.java

```diff
@@ -35,11 +35,12 @@
 //		}
 		
 		List<String> lines = new ArrayList<>();
-		
+		MagicPath path = getPath();
+
 		lines.add("#newsite " + id);
 		lines.add("#level " + level);
 		lines.add("#rarity 5");
-		lines.add("#path " + getPath());
+		lines.add("#path " + path.PathToInteger(path.name));
```

Comment:
This can just be `path.number` (or `getPath().number` if you wanna remove that variable you created)

Reply:
Thanks. Sorry, still working on getting my bearings.